lib/rubocop/cop/rspec/pending_without_reason.rb

```diff
@@ -98,7 +98,9 @@ def on_send(node)
         private
 
         def pending_by_pending_step_without_reason?(node)
-          node.method?(:pending) && node.first_argument.nil?
+          node.method?(:pending) &&
+            node.first_argument.nil? &&
+            inside_example?(node)
```

Comment:
shouldn't we check if we are inside example in the on_send? instead
E.g. having `return unless inside_example?(node)` there

Also do we have a check that it has a `nil` receiver, e.g. that Foo.pending won't produce false positive?

Reply:
>Also do we have a check that it has a nil receiver, e.g. that Foo.pending won't produce false positive?

Indeed it is. I updated this PR.